docfx.json

```diff
@@ -205,6 +213,14 @@
         "*/docs-ref-autogen/postgres/**": [ "Azure Database for PostgreSQL", "Azure", "Azure CLI" ],
         "*/docs-ref-autogen/redis.yml": [ "Cache for Redis", "Azure", "Azure CLI" ],
         "*/docs-ref-autogen/redis/**": [ "Cache for Redis", "Azure", "Azure CLI" ]
+      },
+      "open_source_feedback_issueUrl": {
+         "2017-*/**/*": "https://github.com/Azure/azure-cli/issues",
+         "2018-*/**/*": "https://github.com/Azure/azure-cli/issues",
+         "2019-*/**/*": "https://github.com/Azure/azure-cli/issues",
+         "2020-*/**/*": "https://github.com/Azure/azure-cli/issues",
+         "latest/**/*": "https://github.com/Azure/azure-cli/issues",
```

Comment:
These would preferably point to an issue template, but at a minimum, I would point them to `https://github.com/Azure/azure-cli/issues/new/choose`.